tests/functional/test_pkg_resources_provider.py

```diff
@@ -0,0 +1,92 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+
+# Library imports
+# ---------------
+import copy
+import os
+
+# Third-party imports
+# -------------------
+import pytest
+
+# Local imports
+# -------------
+from PyInstaller.utils.tests import importorskip
+from PyInstaller.compat import exec_python_rc
+
+# :todo: find a way to get this from `conftest` or such
```

Comment:
I think we can tolerate this duplicity. If we do move the modules folder then the worst that can happen if we forget to update this is the tests will fail and we can correct it.

Reply:
OK, I'll remove the comment. It was carried over from an existing test...